testimony/__init__.py

```diff
@@ -73,17 +75,19 @@ class TestFunction(object):
     #: is not available.
     _undefined = object()
 
-    def __init__(self, function_def, parent_class=None):
+    def __init__(self, function_def, parent_class=None, testmodule=None):
         #: A ``ast.FunctionDef`` instance used to extract information
         self.function_def = function_def
         self.parent_class = parent_class
         self.docstring = ast.get_docstring(function_def)
+        self.testmodule = testmodule
         self.assertion = None
         self.bugs = None
         self.feature = None
         self.setup = None
         self.status = None
         self.steps = None
+        self.tags = None
```

Comment:
Not needed. It can stay as string initially.  While doing the tags report, it will be converted to list.